test/OrchardCore.Tests.Modules/BaseThemeSample/BaseThemeSample.csproj

```diff
@@ -6,6 +6,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
Was this just for a safety net, or are you running pack on the whole project folder, rather than just the src folder?

Reply:
I was running pack on the whole project. like was done before ?

Reply:
Ah I just saw it. Everything is packed but only projects under src are kept as artifacts by appveyor: 
https://github.com/OrchardCMS/OrchardCore/pull/7581/files#diff-92ab9a36df5d8e9f7076f2fdec59492d1ac2d9cf27ea046767a7fc4d542ef3dcL90

Reply:
I think we should revert these changes since you are now filtering on the artifacts, and being able to package might be necessary to test the modularity